docs/modules/ROOT/pages/references/component-parameters.adoc

```diff
@@ -231,3 +231,30 @@ default:: ""
 Configures the kubeconfig to use to connect back to the control plane cluster.
 The kubeconfig is only used if the configuration is not in converged mode.
 It has to be created beforehand with the proper RBAC.
+
+== `deploymentManagementSystem`
+[horizontal]
+type:: dict
+
+Controls aspects of the deployment via comp-function revisions.
+
+=== `hotfix`
+[horizontal]
+type:: bool
+default:: false
+
+Defines if all instances should instantly be switched over to the new release or not. This will generate a job that will run the rollout.
```

Comment:
From this comment it's not very clear if the job is generated if the hotfix is enabled or disabled. It also doesn't explain the process if there isn't a job generated.